component/src/main/java/org/wso2/carbon/identity/authenticator/reddit/RedditAuthenticatorConstants.java

```diff
@@ -64,4 +64,7 @@
     public static final String REDDIT_CLIENT_SECRET = "Client Secret";
     //Reddit value for Callback_url
     public static final String REDDIT_CALLBACK_URL = "Callback URL";
+    public static final String OAUTH2_PARAM_ERROR = "error";
+    public static final String ERROR = "error: ";
+    public static final String STATE = ", state: ";
 }
```

Comment:
Please add a new line here

Reply:
Fixed in 712cd7b17f204023cc7106222b39e945853f010f